src/jit/inlinepolicy.h

```diff
@@ -216,13 +216,45 @@ class DiscretionaryPolicy : public LegacyPolicy
     const char* GetName() const override { return "DiscretionaryPolicy"; }
 
 private:
+    
+    void ComputeOpcodeBin(OPCODE opcode);
+    enum { MAX_ARGS = 4 };
```

Comment:
Any reason for 4? Is it just picked up as a small number of arguments that inline policy will track?